src/constants/errors.go

```diff
@@ -289,6 +289,7 @@ const (
 const (
 	GenericJSONErrorMessage = "error"
 	GenericJSONMessage      = "message"
+	ValidationErrors        = "validation errors"
```

Comment:
where is this constant used?

Reply:
I have removed this constant it was unused